server/model.js

```diff
@@ -33,13 +51,25 @@ module.exports = {
         callback(err);
         return;
       }
-      // Grab the results of the query and clean
+      
+      //Format db data to conform to front end [int => string]
+      let cleanObj = {
+        zestimationPrice: singlePropertyData.zestimationprice,
+        thirtyDayPriceChange: singlePropertyData.thirtydaypricechange,
+        oneYearForcast: singlePropertyData.oneyearforcast,
+        comparableHomePrice: singlePropertyData.comparablehomeprice,
+        marketAppreciationPrice: singlePropertyData.marketappreciationprice
```

Comment:
Consider taking advantage of object destructuring to make this code section more readable